etc/categories

```diff
@@ -46,6 +46,8 @@ Deluge|Internet
 Descent 1|Games
 Descent 2|Games
 Deskreen|Internet
+Discord|Internet/Communication
+Discord PTB|Internet/Communication
```

Comment:
applications are added that are not part of this PR and Oh My Posh is not added in the categories file

Reply:
rip switched up branches